index.html

```diff
@@ -276,9 +240,54 @@
           true, the attribute represents a request that the
           <code><a>MediaStream</a></code> object returned as an argument of the
           <code><a>NavigatorUserMediaSuccessCallback</a></code> contains a
-          <a>depth track</a>. If a <a><code>Constraints</code></a> structure is
-          provided, it further specifies the nature and settings of the
-          <a>depth track</a>.
+          <a>depth stream track</a>. If a <a><code>Constraints</code></a>
+          structure is provided, it further specifies the nature and settings
+          of the <a>depth stream track</a>.
+        </p>
+      </section>
+      <section>
+        <h2>
+          <code><a>MediaTrackConstraints</a></code> dictionary
+        </h2>
+        <dl class="idl" title="partial dictionary MediaTrackConstraints">
+          <dt>
+            DepthMapUnit unit = "mm"
+          </dt>
+          <dd>
+            -
+          </dd>
+        </dl>
+        <dl id="enum-basic" class="idl" title="enum DepthMapUnit">
+          <dt>
+            mm
```

Comment:
Millimeter (the American spelling of millimetre) has an SI unit symbol 'mm' so I think that's the best we can do. If we'd use the full name (millimeter) people would probably typo it since the American and International spellings differ subtly.